uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/ArcGISAuthenticationController.cpp

```diff
@@ -22,17 +22,19 @@
 #include <QUrl>
 
 // Maps SDK headers
-#include <ArcGISAuthenticationChallenge.h>
-#include <ArcGISAuthenticationManager.h>
+#include <Authentication/ArcGISAuthenticationChallenge.h>
+#include <Authentication/AuthenticationManager.h>
 #include <ArcGISRuntimeEnvironment.h>
 #include <Error.h>
 #include <ErrorException.h>
-#include <OAuthUserConfiguration.h>
-#include <OAuthUserCredential.h>
-#include <OAuthUserLoginPrompt.h>
-#include <TokenCredential.h>
+#include <Authentication/OAuthUserConfiguration.h>
+#include <Authentication/OAuthUserCredential.h>
+#include <Authentication/OAuthUserLoginPrompt.h>
+#include <Authentication/TokenCredential.h>
```

Comment:
If you merge the latest from v.next into the base `popupElementsWIP` then these differences should go away.

Reply:
I'll give that a shot.

Reply:
I haven't tried merging v.next into the WIP branch just yet.